lib/test_invoke.rb

```diff
@@ -0,0 +1 @@
+puts "hello world"
```

Comment:
I meant to take that out. This was Justin's idea to see if the gem was being invoked.